lib/collections/Projects.js

```diff
@@ -7,6 +7,9 @@ Projects.helpers ({
         amountUsed: value
       }
     })
+  },
+  getImage(){
+    return (this.image === "") ? "http://placehold.it/350x350" : this.image
```

Comment:
Only potential issue I can see is if image is undefined or null. Then it will fail the conditional, but try to return the image anyway.